### PR TITLE
New EOY banners for 2020

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
1) Added new banners for the start of 2020 for the US EOY campaign
2) Cleaned up old banners from the codebase
3) Increased the article count on the global banner to 4 months (per @jessewilkins)

## Does this change need to be reproduced in dotcom-rendering ?
- [X] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## Screenshots
New banners:
With article count:
2020 banner - article count
Without article count:
2020 banner - no article count

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [X] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [X] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [X] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js [hunk 1]
```diff
@@ -3,21 +3,24 @@ import { getSync as geolocationGetSync } from 'lib/geolocation';
 import { acquisitionsBannerUsEoyTemplate } from 'common/modules/commercial/templates/acquisitions-banner-us-eoy';
 import { getArticleViewCountForWeeks } from 'common/modules/onward/history';
 
-// User must have read at least 5 articles in last 4 months (as 17 weeks)
+// User must have read at least 6 articles in last 4 months (as 17 weeks)
 const minArticleViews = 5;
 const articleCountWeeks = 17;
-const articleViewCount = getArticleViewCountForWeeks(articleCountWeeks);
+const articleViewCount = getArticleViewCountForWeeks(articleCountWeeks) - 1;
```
Comment (paulbrown1982) [inline static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: shouldn't `minArticleViews` be 6 and `articleViewCount` remain the same?

Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: The minArticleViewCount is minimum number of articles that will be displayed in the copy, so we are indicating here to never show a lower number than 5. The articleViewCount is the number that is calculated for the reader, that would be rendered in their banner copy. Then we compare them in the canRun function to ensure the number that will be rendered is not lower than the minimum number.

Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: If we did it the way you suggested we could achieve the same result by making minArticleViews equal 6, then not subtract from the articleViewCount, however then we would need to render: `articleViewCount - 1` inside the string, where i don't think any calculations belong

Comment (paulbrown1982) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: Given the worst thing we're doing is telling the reader they've read 1 less article than they actually have, this isn't a big enough issue to warrant any changes at this late stage, so I'll approve.

**Review by paulbrown1982 — APPROVED** [review]
👍 LGTM, question above answered.

File: static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js (hunk 1 above)
Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: no the copy says "You've read more than" which is the need to do this calculation

Comment (paulbrown1982) [reply static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy/contribs-banner-us-eoy-new-year-regulars.js]: Just been directed to the "You've read more than X", which said "You've read X articles" in the previous variant, so the -1 makes total sense now.

**Pull request merged**